win_bin/install/install-dcos-cli.ps1

```diff
@@ -100,6 +102,36 @@ dcos config set package.sources '[\"https://github.com/mesosphere/universe/archi
 
 dcos package update
 
+function AddToPath ($AddedLocation)
+{
+	$Reg = "Registry::HKLM\System\CurrentControlSet\Control\Session Manager\Environment"
+	$OldPath = (Get-ItemProperty -Path "$Reg" -Name PATH).Path
+	$NewPath = $OldPath + ’;’ + $AddedLocation
+	Set-ItemProperty -Path "$Reg" -Name PATH –Value $NewPath
+}
+
+function PromptAddToPath ($AddedLocation)
+{
+	$message = "Modify your Environment to add DCOS to your PATH?"
+	$yes = New-Object System.Management.Automation.Host.ChoiceDescription "&Yes", `
+	    "Yes, add DCOS to PATH."
+	$no = New-Object System.Management.Automation.Host.ChoiceDescription "&No", `
+	    "No, don't add DCOS to PATH."
+	$options = [System.Management.Automation.Host.ChoiceDescription[]]($yes, $no)
+	$result = $host.ui.PromptForChoice("", $message, $options, 0)
+    if ($result -eq 0)
+	{
+		AddToPath "$AddedLocation"
+	}
+}
+
+switch -regex ($add_path)
+{
+  "[Yy].*" {AddToPath "$installation_path\Scripts"; break}
+  "[Nn].*" {break}
+  default {PromptAddToPath "$installation_path\Scripts"}
+}
+
 echo "Finished installing and configuring DCOS CLI."
 echo ""
 echo "Run this command to set up your environment and to get started:"
```

Comment:
Please switch the output here depending on whether or not the path has been altered, the same as the Linux script. You should rebase upon master to pick up the tweaks to this script.